e2e/1_27-exceptions.yml

```diff
@@ -1,10 +1,4 @@
 # This metrics are not present on 1.26
```

Comment:
Can you please fix this comment? It should mention 1.27 instead

Reply:
Done